### DISCUSSION TITLE
Thanks for parenthesize-chained-operators

### DISCUSSION THREAD

**Discussion by boxydog** [astral-sh/ruff#14739] [General]
Between ruff 0.7.2 and 0.8.1 something changed. It looks like maybe parenthesize-chained-operators changed or was finally enforced by default in our pre-commit. (0.8.0 release notes said it "stabilized", not sure what that is.)

Anyway, that re-formatted our code in such a way as to reveal a bug. So, thanks for implementing it!

**Comment by MichaReiser** [comment]
Thank you so much for the kind words. It's great to hear that the new rules are useful to you.
